src/app/globals.css

```
@@ -17,3 +17,36 @@ body {
     )
     rgb(var(--background-start-rgb));
 }
+
+#pln_solda,
+#brk_yuan {
+  position: unset;
+  top: unset;
+  left: unset;
+  z-index: unset;
+  display: none;
+}
+
+#brk_yuan:has(> div) {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  height: 100vh;
+  position: unset;
+  top: unset;
+  left: unset;
+  z-index: unset;
+}
+
+#pln_solda:has(> div) {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  height: 100vh;
+  position: unset;
+  top: unset;
+  left: unset;
+  z-index: unset;
+}
```

```suggestion
#brk_yuan:has(> div), #pln_solda:has(> div) {
  display: flex;
  justify-content: center;
  align-items: center;
  width: 100%;
  height: 100vh;
  position: unset;
  top: unset;
  left: unset;
  z-index: unset;
}
```
Pelo que percebi as 2 classes tem os atributos iguais.

---

nice, é isso mesmo!

---

Done